### PR TITLE
restore the old seach form behaviour

### PR DESCRIPTION
**Pull request by vxsx**
it is necessary to have a template tag, otherwise some variables are not
available anymore. additionaly this fixes a problem with seach input
being focused on touch devices all the time

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

**Comment by vxsx** [comment]
sure

File: CHANGELOG.txt [hunk 1]
```diff
@@ -50,6 +50,8 @@
 - Fixed edit and edit_off constants not being honoured in frontend code
 - Deprecate CMSPlugin.disable_child_plugin in favour of disable_child_plugins
 - Fixed an issue where ``allow_children`` and ``disable_child_plugins`` didn't work on dragitems
+- Replaced search_form template from django admin with a custom on to stop
```
Comment (czpython) [inline CHANGELOG.txt]: with a custom *one*

Comment (vxsx) [inline CHANGELOG.txt]: thanks, fixed

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**